ui/templates/certificate.html

```diff
@@ -10,120 +10,130 @@
+                <a href="http://www.facebook.com/share.php?u={{ request.build_absolute_uri|urlencode }}" target="_blank">
+                    <img src="{% static 'images/certificates/icon-facebook.svg' %}" alt="Share to Facebook">
+                </a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/profile/add?startTask={{ page.product_name|urlencode }}" target="_blank">
+                    <img src="{% static 'images/certificates/icon-linkedin.svg' %}" alt="Share to LinkedIn">
+                </a>
+              </li>
+              <li>
+                <a href="javascript:window.print();">
+                    <img src="{% static 'images/certificates/icon-print.svg' %}" alt="Print">
+                </a>
+              </li>
+            </ul>
+            <h2>Congratulations, {{ name }}!</h2>
+            <p>You have successfully completed {% if course_title %}{{ course_title }}{% else %}{{ program_title }}{% endif %}. Share your accomplishment with your friends, family and colleagues. </p>
+          </div>
+        </div>
       </div>
-
-      <div class="certificate-stamps row">
-        <div class="col-md-3 logo">
-          <a class="edx-logo" href="http://edx.org">
-            <img src="{% static 'images/edx_logo.svg' %}" alt="edX Logo" />
-          </a>
+    </div>
+  {% endif %}
+  <main class="accomplishment accomplishment-main">
+    <div class="wrapper-accomplishment-rendering">
+      <div class="accomplishment-rendering">
+        <div class="wrapper-accomplishment-title verified">
+          <div class="accomplishment-title">
+              <img src="{% static 'images/micromasters_certificate_verified_logo.svg' %}" alt="MicroMaster's Certificate of Completion" />
+              <span class="sr-only">Verified Certificate</span>
+          </div>
+          <div class="wrapper-accomplishment-orgs">
+            <h3 class="accomplishment-orgs-title sr-only">Supported by the following organizations</h3>
+            <ul class="wrapper-orgs list-orgs">
+              <li class="wrapper-organization">
+                <div class="organization">
+                    <img src="{% static 'images/mitx_logo.png' %}" alt="MITx Logo" />
+                </div>
+              </li>
+            </ul>
+          </div>
         </div>
-        <div class="col-md-4 certificate-date">
-          <label>Verified Certificate</label>
-          <span>Issued {{ certificate.created_on|date:"F j, Y" }}</span>
+        <div class="wrapper-accomplishment-statement">
+          <div class="accomplishment-statement">
+            <p class="accomplishment-statement-lead">
+              <span class="accomplishment-statement-detail">This is to certify that</span>
+              <strong class="accomplishment-recipient">{{ name }}</strong>
+              {% if course_title %}
+                <span class="accomplishment-summary">has successfully completed and received a passing grade in</span>
```

Comment:
```suggestion
                <span class="accomplishment-summary">successfully completed and received a passing grade in</span>
```

Reply:
Do we have to change the text for program certificates as well? I have added screenshots in the description you can have a clear comparison there.

Reply:
Yes, we should change them both to match edX.